savoirfairelinux/sous-chef
# Fix #536

## Fixes #536 by lingxiaoyang

### Changes proposed in this pull request:

* Note add view (form, view, template, unittest)
* Client note add view (reused form, view, template, new unittest)

### Status


### How to verify this change

* http://localhost:8000/note/  should have "new note" button.
* http://localhost:8000/note/add/  should work and should redirect back to /note/.
* http://localhost:8000/member/view/612/notes  should have "Add a note" button.
* http://localhost:8000/member/view/612/notes/add  should work, should show the name of client, and should redirect back to client note page.

### Deployment notes and migration

No migration is needed for this change

### New translatable strings


### Additional notes

Previews: https://github.com/savoirfairelinux/sous-chef/issues/536
